cli/dts/lib.deno.ns.d.ts

```diff
@@ -1883,9 +1883,7 @@ declare namespace Deno {
    */
   export const File: typeof FsFile;
 
-  /** **UNSTABLE**: new API, yet to be vetted.
-   *
-   *  @category I/O */
+  /** @category I/O */
   export interface SetRawOptions {
     cbreak: boolean;
```

Comment:
Maybe we should move the docs on the function about this property to instead be on this property?

Reply:
Not sure I understand, can you elaborate?

Reply:
In other words: maybe we should add some docs directly on this property?

Reply:
Done